# ROS timestamp in miliseconds
uint32 timestamp 0

# global ground truth source direction in [-180, 180] degrees
int16 source_direction_deg 

# local ground truth approach angle in [-180, 180] degrees
int16 approach_angle_deg 

# wall distance in meters 
float32 wall_distance_m 

# wall angle in [-180, 180] degrees
int16 wall_angle_deg 
